database.go

```diff
@@ -11,14 +11,29 @@ import (
 
 var db *sql.DB
 
+func checkErrTx(err error, tx *sql.Tx) {
+	if err != nil {
+		tx.Rollback()
+		panic(err)
+	}
+}
+
+func closeTransaction(tx *sql.Tx) {
```

Comment:
Please put this in a separate PR.

Reply:
This means that I have to erase this branch, I don't know how to reverse the actual change in order to separate them, can you help me with some clues?

Reply:
Sorry the above was a question, not an affirmation

Reply:
This means that have I to erase this branch?

Reply:
Let's deny this PR and then you can create two separate ones, then. That's easy.